### ISSUE TITLE
Writing ID3 error - eyeD3/pylast

### ISSUE THREAD

**Issue by j4c3** [automatic-ripping-machine/automatic-ripping-machine#928]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### I have checked the wiki

- [X] I have searched the wiki for this problem

### Does this issue exist in the latest version?

- [X] I'm using the latest release

### Describe the bug?

In the log, after ripping MP3 files, I see the following messages, one for each track:

Writing ID3 version v2.4
-------------------------
/usr/bin/eyeD3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
eyed3.plugins:WARNING: Plugin '('lastfm.py', '/usr/lib/python3/dist-packages/eyed3/plugins')' requires packages that are not installed: No module named 'pylast'

### To Reproduce

I am running the docker container, set the abcde.conf to output MP3 files. 

### What OS are you seeing the problem on?

Other (please state in Anything else? section)

### ARM Release Version or Git hash (You can find this in the arm settings page)

A.R.M version: 2.6.60

### Expected behavior?

No error, I3D info written. 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by 1337-server** [comment]
`requires packages that are not installed: No module named 'pylast'`

Did you try installing `pylast` via pip3 to see if that fixes the issue ?

**Comment by j4c3** [comment]
Yeah, I did that, installed the current version, and the error changes:

/usr/bin/eyeD3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
eyed3.plugins:WARNING: Plugin '('lastfm.py', '/usr/lib/python3/dist-packages/eyed3/plugins')' requires packages that are not installed: cannot import name 'COVER_EXTRA_LARGE' from 'pylast' (/usr/local/lib/python3.8/dist-packages/pylast/__init__.py)

Installing an old pylast, via pip install pylast==2.4.0 does make the error go away.  But, it doesn't seem like the files get ID3 tags written.

**Comment by joelishness** [comment]
I ran into this today.
As a workaround, you can use id3v2 instead of eyeD3 by setting in the config:
ID3TAGV=id3v2.3

**Comment by microtechno9000** [comment]
Can you confirm this is still an issue, looks like eyed3 is installed within ARM
https://github.com/automatic-ripping-machine/arm-dependencies/blame/db6961c1813240585bfac69f38cf6e72b63ac92c/Dockerfile#L55

I am not familiar with the id3 tags and how the pylast package is used

Some additional information would be appreciated

**Comment by rforro** [comment]
Yes, it is still the issue. Yesterday I've installed 2.17.5
